tests/unit/AgreementType.spec.ts

```diff
@@ -37,34 +37,30 @@ const agreementTypeTestCases = [
     'has been completed and a copy has been added to the company\'s record book.',
     customSummaryText: 'The custom Incorporation Agreement and custom Articles ' +
     'has been completed and a copy has been added to the company\'s record book.'
+  }
```

Comment:
It looks like there is a lot of changes in this file but the only change is the CCC test cases are now tested the same as the ULC to reflect CCC's no longer having a sample summary. So it is just some rearranging no new tests were added or deleted.